examples/lazy-loading.html

```diff
@@ -17,31 +17,30 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>&lt;model-viewer&gt; Poster</title>
+  <title>&lt;model-viewer&gt; Lazy Loading</title>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, user-scalable=no,
     minimum-scale=1.0, maximum-scale=1.0">
   <link type="text/css" href="styles/examples.css" rel="stylesheet" />
   <link rel="shortcut icon" type="image/png" href="assets/favicon.png"/>
 
   <!-- The following libraries and polyfills are recommended to maximize browser support -->
-  <!-- Include prismatic.js for Magic Leap support: -->
-  <!--<script src="../node_modules/@magicleap/prismatic/prismatic.min.js"></script>-->
-
-  <!-- Include the Custom Elements ES5 adapter shim if you are loading an IE11-compatible bundle -->
-  <!--<script src="../node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>-->
 
-  <!-- Web Components polyfill is required to support Edge and Firefox < 63: -->
+  <!-- 🚨 REQUIRED: Web Components polyfill to support Edge and Firefox < 63 -->
   <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
 
-  <!-- Intersection Observer polyfill is required for Safari and IE11 -->
+  <!-- 💁 OPTIONAL: Intersection Observer polyfill for better performance in Safari and IE11 -->
   <script src="../node_modules/intersection-observer/intersection-observer.js"></script>
 
-  <!-- Resize Observer polyfill is optional, and improves resize behavior in non-Chrome browsers: -->
+  <!-- 💁 OPTIONAL: Resize Observer polyfill improves resize behavior in non-Chrome browsers -->
   <script src="../node_modules/resize-observer-polyfill/dist/ResizeObserver.js"></script>
 
-  <!-- Fullscreen polyfill is required for using experimental AR features in Canary: -->
+  <!-- 💁 OPTIONAL: Fullscreen polyfill is required for experimental AR features in Canary -->
   <script src="../node_modules/fullscreen-polyfill/dist/fullscreen.polyfill.js"></script>
+
+  <!-- 💁 OPTIONAL: Include prismatic.js for Magic Leap support -->
+  <!--<script src="../node_modules/@magicleap/prismatic/prismatic.min.js"></script>-->
```

Comment:
Should this be included on non-AR examples?

Reply:
No, but it is left here as incidental documentation for those who might be viewing source.